container/multi-value-slice/multi_value_slice.go

```diff
@@ -0,0 +1,380 @@
+package multi_value_slice
```

Comment:
It would be nice to add a package level godoc comment explaining this feature and why we need it. It should also explain a bit about how to use it. How should I initialize a multi value slice ? What should I be aware of when using it? When should I clean it up or destroy it?

Reply:
Very good point. I will add something.